articles/cloud-shell/limitations.md

```diff
@@ -62,6 +62,26 @@ PowerShell in Azure Cloud Shell could take up to 60 seconds to initialize during
 ### No $Home directory persistence
 Data written to $Home by any application (such as: git, vim, and others) does not persist across PowerShell sessions.  For a workaround [see here](troubleshooting.md#powershell-resolutions).
 
+### Update AzureRM error
+When you update the AzureRM module to the lastest version (4.4.0), the following error appears upon startup.
```

Comment:
If you update the AzureRM modules to the latest version (4.4.0), you may see following error on Cloudshell startup